fabric8/src/main/java/org/jboss/arquillian/ce/fabric8/F8OpenShiftAdapter.java

```diff
@@ -487,14 +493,56 @@ private Object triggerBuild(String namespace, String buildName, String secret, S
     }
 
     protected OpenShiftResourceHandle createResourceFromStream(InputStream stream) throws IOException {
-        ModelNode json;
-        try {
-            json = ModelNode.fromJSONStream(stream);
+    	
+    	try {
+	    	StringBuilder textBuilder = new StringBuilder();
+	        try (Reader reader = new BufferedReader(new InputStreamReader
+	          (stream, Charset.forName(StandardCharsets.UTF_8.name())))) {
+	            int c = 0;
+	            while ((c = reader.read()) != -1) {
+	                textBuilder.append((char) c);
+	            }
+	        }
+	        String content = textBuilder.toString();
+	       
```

Comment:
Could we save a few lines of code here and use something like `String theString = IOUtils.toString(inputStream, encoding);` - http://stackoverflow.com/questions/309424/read-convert-an-inputstream-to-a-string